torch/_inductor/decomposition.py

```diff
@@ -22,13 +22,52 @@
         aten.linalg_vector_norm,
         aten.std,
         aten.std_mean,
-        aten._to_copy,
         aten.triu_indices,
         aten.tril_indices,
         aten.sqrt_,
         aten.lcm,
         aten.clamp_min_,
         aten.sin_,
+        aten.addcdiv,
```

Comment:
most (if not all) of these additions should go to core_aten_decompositions in _decomp/__init__.py, cc @SherlockNoMad

Reply:
Right, I wasn't sure what was the reasoning for something going here vs `core_aten_decompositions`. What's the idea there?

Reply:
`core_aten_decompositions` is for decompositions that do aten->aten (don't lower to prims). They can be used by backends that support aten ops only, no prims.